niche_vlaanderen/system_tables/management.csv

```diff
@@ -1,4 +1,4 @@
-code,management,influence
+management,description,influence
```

Comment:
You added a dutch description, but used the english column name

Reply:
translation in English added, next to Dutch.